shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/Server.java

```diff
@@ -468,6 +469,9 @@ public String activeUrl() {
               public Stream<Header> headers() {
                 final Instant now = Instant.now();
                 final ObjectNode searchInfo = RuntimeSupport.MAPPER.createObjectNode();
+                pluginManager
+                    .searches(FilterBuilder.JSON)
+                    .forEach(pair -> searchInfo.putPOJO(pair.first(), pair.second()));
```

Comment:
Fun bug while testing, the UI expects a list of filters. Can you make `pair.second()` into a singleton list?

Reply:
Actually, no, that fails to add the `type` attribute. This works:
```
.forEach(pair -> searchInfo.putArray(pair.first()).addPOJO(pair.second()));
```